.github/workflows/release-build.yml

```diff
@@ -0,0 +1,85 @@
+name: "Release Build"
+
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: windows-2022
+    strategy:
+      matrix:
+        Game: 
+          - Ares
+          - TS
+          - YR
+    
+    outputs:
+      GitVersion_SemVer: ${{ steps.set_gitversion_outputs.outputs.GitVersion_SemVer }}
```

Comment:
Forgot to use this output?

Reply:
No. It is used via `needs.build.outputs.GitVersion_SemVer`